README.md

```diff
@@ -58,34 +58,51 @@ integrates with the Smartling translation platform.
     }
     ```
 
-If your project locales do not match those in Smartling (e.g. `ro` in your project, `ro-RO` in Smartling),
-then you can provide a Wagtail locale id to Smartling locale id mapping via the `LOCALE_TO_SMARTLING_LOCALE` setting:
+    If your project's locales do not match those in Smartling (e.g. `ro` in your
+    project, `ro-RO` in Smartling), then you can provide a Wagtail locale ID to
+    Smartling locale ID mapping via the `LOCALE_TO_SMARTLING_LOCALE` setting:
 
-```python
-WAGTAIL_LOCALIZE_SMARTLING = {
-    "LOCALE_TO_SMARTLING_LOCALE": {
-        "ro": "ro-RO"
+    ```python
+    WAGTAIL_LOCALIZE_SMARTLING = {
+        "LOCALE_TO_SMARTLING_LOCALE": {
+            "ro": "ro-RO"
+        }
     }
-}
-```
+    ```
 
-or you can specify a callable or a dotted path to a callable in the `LOCALE_MAPPING_CALLBACK` setting
+    ... or you can specify a callable or a dotted path to a callable in the
+    `LOCALE_MAPPING_CALLBACK` setting:
 
-```python
-def map_project_locale_to_smartling(locale: str) -> str:
-    if locale == "ro":
-        return "ro-RO"
-    return locale
+    ```python
+    def map_project_locale_to_smartling(locale: str) -> str:
+        if locale == "ro":
+            return "ro-RO"
+        return locale
 
 
-WAGTAIL_LOCALIZE_SMARTLING = {
-    # ...
-    "LOCALE_MAPPING_CALLBACK": "settings.map_project_locale_to_smartling"
-}
-```
+    WAGTAIL_LOCALIZE_SMARTLING = {
+        # ...
+        "LOCALE_MAPPING_CALLBACK": "settings.map_project_locale_to_smartling"
+    }
+    ```
+
+    The callback receives a `WAGTAIL_CONTENT_LANGUAGES` local code string and is
+    expected to return a valid mapped locale ID (or the original locale ID).
+
+    Note that by default, when syncing translations the project will attempt to
+    reformat a mixed-case, Smartling-style language code (e.g. `zh-CN`) into a
+    Django-style all-lowercase code (e.g. `zh-cn`). Depending on how languade
```

Comment:
```suggestion
    Django-style all-lowercase code (e.g. `zh-cn`). Depending on how language
```

This may have been me. Other than this, :shipit: !